website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,113 @@
+| `createIndexes`           | string   | The name of the collection.                                                                                                                                                                                           |
+| `indexes`                 | array    | An array of index specification documents. Each document defines an index.                                                                                                                                            |
+| `key`                     | document | A single document that contains the field(s) to index and their sort order or type (`1`, `-1`, `"text"`, `cosmosSearch` for vector indexes, etc).                                                                     |
+| `name`                    | string   | A required name for the index. Must be unique to the collection.                                                                                                                                                      |
+| `unique`                  | boolean  | Ensures all values are unique for the indexed field if set to `true`.                                                                                                                                                 |
+| `partialFilterExpression` | document | Restricts index to documents matching the condition.                                                                                                                                                                  |
+| `expireAfterSeconds`      | int32    | For TTL indexes. Automatically deletes documents after set time.                                                                                                                                                      |
+| `sparse`                  | boolean  | Only indexes documents that contain the indexed field if set to `true`.                                                                                                                                               |
+| `cosmosSearchOptions`     | document | Specifies the configuration parameters (`kind`, `similarity`, `dimensions`, etc.) for the vector index to suit your specific use case. [See vector index parameters for more details](../../guides/vector-search.mdx) |
+
+## Example usage
+
+The following examples demonstrate how to create different types of indexes.
+
+### Create an index
+
+This example creates a simple index on the `title` field in the `books` collection.
+
+<CodeBlock language="js">{CreateIndex}</CodeBlock>
+
+### Create compound index
+
+This example creates a compound index on the `publication.date` and `analytics.average_rating` fields in the `books` collection.
+
+<CodeBlock language="js">{CreateCompoundIndex}</CodeBlock>
```

Comment:
I wonder if there is way to make CTS to generate multi line for instead of scrolling :thinking:

Reply:
Great, let's regenerate CTS again please. In CI, it generates multi line version :hugs: